### PR TITLE
Changes in filter on catalog page

### LINKED ISSUE #1185: Changes in filter on catalog page for mobile devices
### Description
As we are not rendering cards for all option for mobile devices it's better if we remove 'All' option from catalog page for mobile devices.

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Savio629**
**Description**
Changes in filter on catalog page for mobile devices

This PR fixes #1185 

**Notes for Reviewers**
Draft pr

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Savio629** [comment]
@vishalvivekm @thisiskaransgit Do we need to add 'View all' Button?

**Comment by Savio629** [comment]
Ummm build is failing.....

For the meantime: 

https://github.com/meshery/meshery.io/assets/91362589/96dd1186-6224-484f-a7a5-e667305e1a56

**Comment by Savio629** [comment]
@thisiskaransgit why is build still failing?

**Comment by thisiskaransgit** [comment]
@Savio629, Thank you for working on this, this was put on hold to complete the integration update task. We can move now with this one. Kindly resolve the merge conflicts.

**Comment by Savio629** [comment]
@thisiskaransgit Will resolve the conflicts and update you by the end of the day or mostly by tomorrow 👍

**Comment by Savio629** [comment]
@thisiskaransgit

**Review by GaganpreetKaurKalsi — CHANGES_REQUESTED** [review]
Hey @Savio629, 
1. I see these characters at the right side of the screen
image

2. Also, we might want to have `cursor: pointer;` for "View All" button.
3. Maybe decrease of spacing between filters and also have them in a single column instead of 2 as in phones with smaller screen widths, the content is overflowing. Or if we want to have 2 columns, it should become one when the content overflows. 
image

<br><br>

**Let's wait to hear from others as well on point 3.**

**Comment by Savio629** [comment]
Fixed 1st and 2nd issue 👍
Regarding the 3rd issue, there wasn't any issue earlier but as I pulled the new changes just saw that it was not coded in such a way to avoid overflowing,I'll fix it..
I think as screen width decreases it should be in a single column

**Comment by Savio629** [comment]
@GaganpreetKaurKalsi I have made the changes regarding the 3rd point
Also there are two deployment categories

**Comment by GaganpreetKaurKalsi** [comment]
> // @thisiskaransgit @GaganpreetKaurKalsi Any other changes required?


See the first row of Cards, it is being cut  @Savio629 
image

**Comment by Savio629** [comment]
@GaganpreetKaurKalsi Done👍

**Review by GaganpreetKaurKalsi — APPROVED** [review]
Looks good @Savio629 👍

File: _layouts/catalog.html [hunk 1]
```diff
@@ -21,5 +21,6 @@ <h3>Pattern Management</h3>
   {% include patterns-filter.html %}
   {{ content }}
 </div>
-  <!-- {% include card-modal.html %}   -->
+  {% include card-modal.html %}
```
Comment (thisiskaransgit) [inline _layouts/catalog.html:24]:
```suggestion
  <!-- {% include card-modal.html %}   -->
```

**Review by thisiskaransgit — APPROVED** [review]

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

File: _layouts/catalog.html (hunk 1 above)
Comment (thisiskaransgit) [reply _layouts/catalog.html:24]: @Savio629, keep them disabled

**Comment by Chadha93** [comment]
Merge conflicts @Savio629 ...

**Comment by Chadha93** [comment]
@vishalvivekm A good one for your :eyes:

**Comment by Chadha93** [comment]
@Savio629 Heads up merge conflicts...

**Comment by Savio629** [comment]
// @thisiskaransgit

**Comment by leecalcote** [comment]
@Savio629, you'll need to coordinate with @Chadha93. @thisiskaransgit is inactive in the community and project.

**Comment by Chadha93** [comment]
@vishalvivekm Will you review this PR? I asked at our last meeting too. 

I'll review this tonight. cc @Savio629

**Comment by Savio629** [comment]
@Chadha93 @vishalvivekm

**Comment by leecalcote** [comment]
Thanks, @Savio629. 

The formatting of the cards in mobile view could use you help.

**Pull request merged**